Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -3056,6 +3056,16 @@ - (void)_beginLiveResize
     [self _ensureResizeAnimationView];
 }
 
+- (void)_doAfterVideoFullscreenPlayerLayerBoundsHaveChanged:(void (^)(void))updateBlock
+{
+    _page->callAfterVideoFullscreenPlayerLayerBoundsHaveChanged([updateBlockCopy = makeBlockPtr(updateBlock), strongSelf = retainPtr(self)] {
```

Comment:
do I actually need to capture self?

Reply:
I don't see why self needs to be captured, it is used used